tests/integration/test_repos_repo.py

```diff
@@ -426,6 +426,16 @@ def test_release(self):
 
         assert isinstance(release, github3.repos.release.Release)
 
+    def test_release_by_tag_name(self):
+        """Test the ability to retrieve a release by tag name"""
+        cassette_name = self.cassette_name('release')
```

Comment:
yes i want  to do that but i still a json file to use as a cassette....how can i do that?